schemas/messages.json

```diff
@@ -28,13 +28,22 @@
     "frameworkStates": {
       "properties": {
         "coming": {
-          "type": "object"
+          "type": ["object", "string"]
```

Comment:
Does our code know what to do when handed an `object` in this field instead of a `string`?

Reply:
It does in the places where it expects an object.

Reply:
More specifically - these "frameworkStates" keys are used for a couple of different sets of messages: for the `homepage-sidebar` messages our frontend is expecting an object and knows what to do with it; the new `become-a-supplier` messages just have the straight-up string.
I think it's fine that the frontend will die if it gets the wrong thing, as I said in the commit message here: https://github.com/alphagov/digitalmarketplace-frameworks/pull/469/commits/bed9dc07c7416a387d24dced4d36a23c11022f55

Reply:
Hmmmm okaaay, would seem it would be better to have it be a single-item dict in the simple case with the extra properties being optional but whatevs.